docs/topics/Migrations.md

```diff
@@ -30,54 +42,104 @@ dotnet ef migrations add [Entity_ChangesMade] \
 
 ## Removing Migrations
 
-If you need to undo a migration, run the following command to revert the most recent migration.
-
-```Shell
-dotnet ef migrations remove \
---project Database \
---startup-project API \
---context OtrContext
-```
+<tabs group="os">
+   <tab id="Windows-remove" title="Windows" group-key="Windows">
+      If you need to undo a migration, run the following command to revert the most recent migration.<br/>
+      <code-block>
+      dotnet ef migrations remove `
+      --project Database `
+      --startup-project API `
+      --context OtrContext
+      </code-block>
+   </tab>
+   <tab id="Else-remove" title="Linux &amp; macOS" group-key="Else">
+      If you need to undo a migration, run the following command to revert the most recent migration.<br/>
+      <code-block>
+      dotnet ef migrations remove \
+      --project Database \
+      --startup-project API \
+      --context OtrContext
+      </code-block>
+   </tab>
+</tabs>
 
 ## Applying Migrations
 
-Run the following to apply any pending migrations to the database. In development, ensure the database appears as you expect it to after applying migrations.
-
-```Shell
-dotnet ef database update \
---project Database \
---startup-project API \
---context OtrContext
-```
+<tabs group="os">
+   <tab id="Windows-apply" title="Windows" group-key="Windows">
+      Run the following to apply any pending migrations to the database. In development, ensure the database appears as you expect it to after applying migrations.<br/>
```

Comment:
```suggestion
      Run the following to apply any pending migrations to the database. In development, ensure the database appears as expected after applying migrations.<br/>
```

Best to avoid first/second person